docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+
+<KDropdownMenu label="Documentation" :items="deepClone(defaultItemsUnselected)" />
+
+## Props
+
+### label
+
+The label for the menu.
+
+### items
+
+An array of items containing a `label` and other optional properties will render a menu of [`KDropdownItems`](#KDropdownItem) .
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :items="[
+    { label: 'Home', to: { path: '/' } },
+    { label: 'Button docs', to: { path: '/components/button.html' } },
+    { label: 'My docs', to: { path: '/components/dropdown-menu.html' } }
+  ]"
+/>
+```
+
+### appearance
+
+Use this prop to specify the display style for the dropdown menu. Can be either `menu` (default) or `selectionMenu`.
+The `menu` style is the standard you have seen in the example above.
+
+The `selectionMenu` style is good for a clearer representation of what is selected. `selected` state is handled automatically when
+clicking a `KDropdownItem` if used in conjunction with the `items` prop. You will need to manually control selectedness if you are using the `items` slot.
+
+<div>
+  <KDropdownMenu
+    :kpop-attributes="{ width: '220' }"
+    :label="selectedLabel"
+    appearance="selectionMenu"
+  >
+    <template #items>
+      <KDropdownItem
+        :selected="selectedItem === 'us'"
+        @click="clickHandler('US selected', 'us', 'US (United States)')"
+      >
+        US (United States)
+      </KDropdownItem>
+      <KDropdownItem
+        :selected="selectedItem === 'fr'"
+        @click="clickHandler('France selected', 'fr', 'FR (France)')"
+      >
+        FR (France)
+      </KDropdownItem>
+    </template>
+  </KDropdownMenu>
+</div>
+
+```html
+<KDropdownMenu
+  :kpop-attributes="{ width: '220' }"
+  :label="selectedLabel"
+  appearance="selectionMenu"
+>
+  <template #items>
+    <KDropdownItem
+      :selected="selectedItem === 'us'"
+      @click="clickHandler('US selected', 'us', 'US (United States)')"
+    >
+      US (United States)
+    </KDropdownItem>
+    <KDropdownItem
+      :selected="selectedItem === 'fr'"
+      @click="clickHandler('France selected', 'fr', 'FR (France)')"
+    >
+      FR (France)
+    </KDropdownItem>
+  </template>
+</KDropdownMenu>
+
+<script>
+export default {
+  data() {
+    return {
+      selectedItem: '',
+      selectedLabel: 'Select an item'
+    }
+  },
+  methods: {
+    clickHandler (msg, val, label) {
+      if (val !== undefined) {
+        this.selectedItem = val
+      }
+
+      if (label) {
+        this.selectedLabel = label
+      }
+
+      this.$toaster.open(msg)
+    }
+  }
+}
+</script>
+```
+
+### showCaret
+
+Use this prop if you would like the trigger button to display the caret.
+
+<KDropdownMenu label="Documentation" :items="deepClone(defaultItemsUnselected)" show-caret />
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :items="items"
+  show-caret
+/>
+```
+
+### kpopAttributes
+
+Use the `kpopAttributes` prop to configure the **KPop** [props](/components/popover.html) dropdown.
+
```

Comment:
I'm not sure the card example is ideal here since the Docs user isn't familiar with the "three dot" action menu UI and may not understand this example without clicking around or some other instruction?

Reply:
Fixed - added instruction

![image](https://user-images.githubusercontent.com/67973710/182684007-202c7c8b-c1f3-406c-9a68-0d25c4f510bb.png)